src/hotspot/share/classfile/altHashing.cpp

```diff
@@ -110,7 +110,7 @@ static void halfsiphash_adddata(uint32_t v[4], uint32_t newdata, int rounds) {
 
 static void halfsiphash_init32(uint32_t v[4], uint64_t seed) {
```

Comment:
I believe all the bitwise operators here will promote the result to int. Will this cause further conversion issues with uint32_t?

Reply:
Since uint32_t is wider, I don't think so (at least I don't get a warning for it with -Wconversion).